.github/workflows/release.yml

```diff
@@ -74,13 +74,39 @@ jobs:
 
   build-debian:
     name: Build Debian packages
+    runs-on: ubuntu-latest
     needs: tag
-    uses: eclipse-zenoh/ci/.github/workflows/build-crates-debian.yml@main
-    with:
-      repo: ${{ github.repository }}
-      version: ${{ needs.tag.outputs.version }}
-      branch: ${{ needs.tag.outputs.branch }}
-    secrets: inherit
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.tag.outputs.branch }}
+          token: ${{ secrets.BOT_TOKEN_WORKFLOW }}
+
+      - id: build
+        run: |
+          mkdir -p build  && cd build
+          cmake ..
```

Comment:
I believe it should be
```
cmake .. -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release -DZENOHC_BUILD_WITH_UNSTABLE_API=ON -DZENOHC_BUILD_WITH_SHARED_MEMORY=ON
```

since we want a shared lib with all the features